docs/user/trusted-publishers/security-model.md

```diff
@@ -258,6 +280,10 @@ own security model and considerations.
       access the OIDC token to a bare minimum. This prevents both accidental
       and malicious disclosure.
 
+[played a major role in cyber attacks]: https://verizon.com/dbir
```

Comment:
This HTTPS link doesn't connect successfully for me, but the HTTP version redirects to <https://www.verizon.com/business/resources/reports/dbir/>. Perhaps we should update it to that?

Reply:
Looks like the actual report is [here](https://www.verizon.com/business/resources/reports/2024-dbir-data-breach-investigations-report.pdf), but it's emailwalled. I think we should probably find an alternative source/cite, since PyPI's docs shouldn't link people to walled sources 🙂 

Here are some candidates:

* <https://therecord.media/cisa-cyberattacks-using-valid-credentials>
* <https://www.cisa.gov/sites/default/files/2023-07/FY22-RVA-Analysis%20-%20Final_508c.pdf>

Reply:
The email-walled version is unfortunately the one the authors ask that we link to. But that first link looks good, will replace it.